### PR TITLE
DBZ-1320 documentation update for outbox event router

### PR DESCRIPTION
**Pull request by brbrown25**

### CODE REVIEW

File: docs/configuration/outbox-event-router.asciidoc [hunk 1]
```diff
@@ -141,6 +141,12 @@ This result was achieved with the link:#default-table-columns[default configurat
 |Router
 |The name of the topic in which the events will be routed, a replacement `${routedByValue}` is available which is the value of The column configured via `route.by.field`
 
+|`route.tombstone.on.empty.payload`
+|`false`
+|Router
+|Whether or not an empty payload should cause a tombstone event.
```
Comment (gunnarmorling) [inline docs/configuration/outbox-event-router.asciidoc]:
```suggestion
|Whether or not an empty or `null` payload should cause a tombstone event.
```

**Comment by gunnarmorling** [comment]
Thanks, @brbrown25! Could you only rework the commit using an e-mail address associated to your GitHub account?

**Comment by brbrown25** [comment]
I will make the suggested change and update the commit email this evening!

**Comment by brbrown25** [comment]
@gunnarmorling update email and made the proposed text change!

**Review by renatomefi — APPROVED** [review]
Very nice!

**Comment by gunnarmorling** [comment]
Yes, thanks a lot, @brbrown25! This will be merged once the code change gets out as part of the next release.

**Review by gunnarmorling — APPROVED** [review]

**Comment by cmd-ninja** [comment]
Nice!

**Pull request merged**